scripts/hera-sim-vis.py

```diff
@@ -132,6 +136,16 @@ def cprint(*args, **kwargs):
         for fnc in simulator._functions_to_profile:
             profiler.add_function(fnc)
 
+        # Now add any user-defined functions that they want to be profiled.
+        # Functions must be sent in as "path.to.module:function_name" or
+        # "path.to.module:Class.method".
```

Comment:
Can you add the part about the required format to the `help` info for the `profile_funcs` argument?

Reply:
Ah yeah, good idea.